cffi.asd

```diff
@@ -65,11 +65,28 @@
      (:file "foreign-vars")
      (:file "features")))))
 
-(defmethod operation-done-p ((o test-op) (c (eql (find-system :cffi))))
-  nil)
-
-(defmethod perform ((o test-op) (c (eql (find-system :cffi))))
-  (operate 'asdf:load-op :cffi-tests)
-  (operate 'asdf:test-op :cffi-tests))
```

Comment:
it's a mistake, i have assumed that a functionality we use is in asdf, but it's in fact in our own lib... i'll revert this partially (the operation-done-p method is not needed anymore)